tests/msc3902/federation_room_join_partial_state_test.go

```diff
@@ -4080,8 +4080,8 @@ func TestPartialStateJoin(t *testing.T) {
 		}
 		t.Log("Alice begins a partial join to a room")
 		alice := deployment.Register(t, "hs1", helpers.RegistrationOpts{
-			Localpart: "t46alice",
-			IsAdmin:   true,
+			LocalpartSuffix: "t46alice",
+			IsAdmin:         true,
```

Comment:
Would be very surprised if alice actually needs Admin here, similar for terry above. Te `t.Log()` makes it look like I wrote this, so maybe I was completely off my rocker here.

Reply:
Agreed, this surprised me but also I wasn't about to touch it.